### PR TITLE
Http request support

### PR DESCRIPTION
**Pull request by ktrance**
Added support for accessing basic http request information.

This pull request adds basic http request support in flat .Net objects adding no further dependencies.

### CODE REVIEW

File: environments/dotnet/README.md [hunk 1]
```diff
@@ -163,6 +163,60 @@ $ curl "http://$FISSION_ROUTER/add?x=30&y=12"
...
+
+Secondly you need to create a file /tmp/func.cs containing the following code:
+
+```
+using System;
+using Fission.DotNetCore.Api;
+
+public class FissionFunction
+{
+    public string Execute(FissionContext context){
```
Comment (soamvasani) [inline environments/dotnet/README.md]: Could you also add an example for reading out context.Request.Body?

**Comment by soamvasani** [comment]
Nice!  Would you mind creating a directory for C# dotnet under examples/ and adding one or two examples there?

File: environments/dotnet/README.md (hunk 1 above)
Comment (ktrance) [inline environments/dotnet/README.md]: Will do, give me a day or two :)

Comment (ktrance) [inline environments/dotnet/README.md]: I have added sample on how to deserialize a json request body into objects and I have added a convenience method that will give you the request body as a string

**Comment by soamvasani** [comment]
Looks great, thanks again! Merging now.

**Pull request merged**